README.md

```diff
@@ -57,28 +57,24 @@ build and push an image for the operator to
 ## Create a CatalogSource for [OLM](https://github.com/operator-framework/operator-lifecycle-manager)
 
 The OLM requires special manifests that the operator-sdk can help
-generate. First we extract the CRD's from the upstream manifest[s]
-contained within [deploy/resources](deploy/resources/):
+generate.
 
-    ./hack/extract-crds.sh
+Create a `ClusterServiceVersion` for the version that corresponds to
+those manifest[s] beneath [deploy/resources](deploy/resources/). The
+`$PREVIOUS_VERSION` is the CSV yours will replace.
 
-Update [deploy/role.yaml](deploy/role.yaml/) to match the RBAC
-policies in the upstream manifest[s].
+    operator-sdk olm-catalog gen-csv \
+        --csv-version $VERSION \
+        --from-version $PREVIOUS_VERSION
 
-And then generate a basic `ClusterServiceVersion` passing in a version
-that corresponds to those manifests:
-
-    operator-sdk olm-catalog gen-csv --csv-version 0.5.1
-
-Some post-editing of the file it generates is required:
+Most values should carry over, but if you're starting from scratch,
+some post-editing of the file it generates will be required:
 
 * Add fields to address any warnings it reports
-* Add `description` and `displayName` fields for all owned CRD's
-* Add `args: ["--olm", "--install"]` to the operator's container spec.
-* Add a `replaces` field referencing the previous CSV
+* Verify `description` and `displayName` fields for all owned CRD's
```

Comment:
Wondering: I see only one _owned_ CRD, the `kind: Install` - which comes from the operator (`serving_v1alpha1_install_crd.yaml`). could be added there ? 

Is that the plan, that there is only "own" CRDs, as owned (by the operator). previously we added all upstream bits that, marking them as `owned` .

Reply:
Not sure if `gen-csv` will honor those field or not. But re only having that CRD, yes, I think so. Seeing all the upstream ones in the OLM console seems a little overwhelming to the user, I think, but we can revisit that. For now, I like not having to extract them and keep them in sync across versions. And the `Install` is enough to express dependencies between operators if need be.